docs/source/reference/storage.rst

```diff
@@ -1,303 +1,291 @@
-      # mode. This means instead of copying contents of the remote bucket to the VM,
-      # sky "mounts" the bucket at /dataset-storage-mount. Files are streamed from
-      # S3 as they are read by the task. Any writes made at /dataset-storage-mount
-      # are also replicated on the remote S3 bucket and any other storage mounts
-      # using the same bucket with MOUNT mode. Note that the source is synced with
-      # the remote bucket everytime this task is run.
-      /dataset-storage-mount:
-        name: sky-dataset-romil
-        source: ~/datasets
-        mode: MOUNT
-
-      # *** Mounting very large public buckets ***
-      #
-      # This uses the MOUNT mode to mount a mount at 3.5 TB public bucket at the
-      # specified path. Since MOUNT mode is used, the bucket is not copied at init,
-      # instead contents are streamed from S3 as they are requested. This saves disk
-      # space on the remote VM.
-      # Since this is a public bucket, any writes to the path will fail.
-      /huge-dataset-mount:
-        source: s3://digitalcorpora
-        mode: MOUNT
-
-      # *** Collecting outputs of tasks on S3 ***
-      #
-      # This uses the MOUNT mode to create an output mount path. This creates an
-      # empty bucket with the specified name and mounts it at the path.
-      # Any files written to /outputs-mount will also be synced to my-output-bucket.
-      # This is useful when you want to collect outputs of your task directly in a
-      # S3 bucket and browse it from your laptop later.
-      #
-      # Since writes are synced across workers mounting the same bucket,
-      # this approach can also be used to create a shared filesystem across workers.
-      # See examples/storage/pingpong.yaml for an example.
-      /outputs-mount:
-        name: my-output-bucket
-        mode: MOUNT
-
-      # *** Uploading multiple files to the same Storage object ***
-      #
-      # The source field in a storage object can also be a list of local paths.
-      # This is useful when multiple files or directories need to be uploaded to the
-      # same bucket.
-      #
-      # Note: The basenames of each path in the source list are copied recursively
-      # to the root of the bucket. Thus, if the source list contains a directory,
-      # the entire directory is copied to the root of the bucket. For instance,
-      # in this example, the contents of ~/datasets are copied to
-      # s3://sky-multisource-storage/datasets/. ~/mydir/myfile.txt will appear
-      # at s3://sky-multisource-storage/myfile.txt.
-      /datasets-multisource-storage:
-        name: sky-multisource-storage # Make sure this name is unique or you own this bucket
-        source: [~/mydir/myfile.txt, ~/datasets]
-
-
-    run: |
-      pwd
-      ls -la /
-
-    # Remember to run `sky storage ls` and `sky storage delete` to delete the
-    # created storage objects!
+    .. tab-item:: Create (or re-use) a bucket
```

Comment:
Several options

1. "Use an existing bucket" / "Create and reuse a bucket" ("_and_" should avoid the conflict?)
1. "Use an external bucket" / "Create and reuse a bucket" 
2. "Use an existing bucket" / "Create a bucket"

Slight preference for 1 & 2, since I think the "reuse" part may be common to warrant being in the tabs; that said,  3 is ok with me too if @romilbhardwaj and @Michaelvll agree :)

Reply:
I'm fine with 2 or 3

Reply:
I would prefer 2 : )